### ISSUE TITLE
Livereload functionality does not appear to work

### ISSUE THREAD

**Issue by kevinold** [wlepinski/generator-meanstack#13]
I've generated a project with 0.1.0 of this generator and run "grunt server".

I receive the following output:

```
Running "server" task

Running "clean:server" (clean) task

Running "concurrent:server" (concurrent) task

    Running "coffee:dist" (coffee) task

    Done, without errors.

Running "express:livereload" (express) task

Running "open:server" (open) task

Running "watch" task
Waiting...OK
```

Then a browser window is opened going to http://localhost:3000/, but in Chrome I receive the "Oops!" page in that it can't load the page as the server isn't started.

It is about 1 second later that this appears in the terminal (note: after the browser was launched by "watch" livereload):

```
Running "express-server:livereload:/var/folders/84/r6x7y3214tl81brhvjqyfpt00000gn/T/express1131120-42197-1hvbtcz/livereload.server" (express-server) task
Web server started on port:3000, no hostname specified [pid: 42200]

Running "express-keepalive" task
```

If I update app/index.html it does not reload trigger an update in the browser.  I have tried CSS and JS files defined in the watch as well as the express server and EJS template files (even though it isn't defined).

I have not been able to find a cause for this after trying to rearrange the order in which "open", "watch" and "express:livereload" were defined in the "server" task.

**Comment by rovens** [comment]
I've experienced the same problem. As a workaround I have modified the Gruntfile to allow the server to start by commenting out the serverreload:true statement. 

```
express: {
      options: {
        port: 3000,
        hostname: '*'
      },
      livereload: {
        options: {
          livereload: true,
          //serverreload: true,
          server: path.resolve('app.js'),
          bases: [path.resolve('./.tmp'), path.resolve(__dirname, yeomanConfig.app)]
        }
      },
```

**Comment by DaveNascimento** [comment]
That did the job for me, thanks @rovens!

**Comment by sebkouba** [comment]
@wlepinski Maybe disable the serverreload as default? Otherwise people end up with a non working scaffolding right away which is discouraging.

**Comment by skywritergr** [comment]
Thanks @rovens that worked for me too. I've been wondering for the last couple of days why it doesn't work.

**Comment by wlepinski** [comment]
Do you guys think that would be nice to disable the serverreload by default until we find a good alternative to it?

**Comment by wlepinski** [comment]
Closed on 74129605
